api/tests/unit/environments/test_unit_environments_views_sdk_environment.py

```diff
@@ -44,11 +46,37 @@ def test_get_environment_document(
             rule=nested_rule,
         )
 
+        # Let's create segment override for each segment too
+        feature_segment = FeatureSegment.objects.create(
+            segment=segment, feature=feature, environment=environment
+        )
+        FeatureState.objects.create(
+            feature=feature,
+            environment=environment,
+            feature_segment=feature_segment,
+            enabled=True,
+        )
+
+    for i in range(10):
+        mv_feature = Feature.objects.create(
+            name=f"mv_feature_{i}", project=project, type=MULTIVARIATE
+        )
+        MultivariateFeatureOption.objects.create(
+            feature=mv_feature,
+            default_percentage_allocation=10,
+            string_value="option-1",
+        )
+        MultivariateFeatureOption.objects.create(
+            feature=mv_feature,
+            default_percentage_allocation=10,
+            string_value="option-2",
+        )
+
     # and the relevant URL to get an environment document
     url = reverse("api-v1:environment-document")
 
     # When
-    with django_assert_num_queries(11):
+    with django_assert_num_queries(13):
```

Comment:
This number increased to 23 before I added the select related. I think the additional 2 queries are legitimate for retrieving the MV data and the segment overrides.